### PR TITLE
Fix a number of completion issues

### PR DESCRIPTION
**Pull request by vasily-kirichenko**
It fixed this
image

this 

image

and this

image

### CODE REVIEW

File: src/fsharp/vs/ServiceDeclarations.fs [hunk 1]
```diff
@@ -1397,14 +1397,14 @@ type FSharpDeclarationListInfo(declarations: FSharpDeclarationListItem[]) =
...
-                | [Item.MethodGroup(_,[_],_)] -> IsOperatorName name
+                | [Item.MethodGroup _ ] -> IsOperatorName name
                 | [Item.UnionCase _] -> IsOperatorName name
                 | _ -> false              
 
             let isFSharpList name = (name = "[]") // list shows up as a Type and a UnionCase, only such entity with a symbolic name, but want to filter out of intellisense
 
             items |> List.filter (fun (name, items) -> not (isOperatorItem(name, items)) && not (isFSharpList name)) 
-
+
```
Comment (Phrohdoh) [inline src/fsharp/vs/ServiceDeclarations.fs]: Unnecessary whitespace.

File: src/fsharp/vs/ServiceUntypedParse.fs [hunk 2]
```diff
@@ -1171,5 +1171,10 @@ module UntypedParseImpl =
                             | None -> Some (CompletionContext.Invalid) // A $ .B -> no completion list
                         | _ -> None 
                         
-                    member this.VisitBinding(defaultTraverse, synBinding) = defaultTraverse synBinding }
+                    member this.VisitBinding(defaultTraverse, synBinding) = defaultTraverse synBinding 
+
```
Comment (Phrohdoh) [inline src/fsharp/vs/ServiceUntypedParse.fs]: Unnecessary whitespace on these lines.

File: src/fsharp/vs/ServiceDeclarations.fs (hunk 1 above)
Comment (vasily-kirichenko) [inline src/fsharp/vs/ServiceDeclarations.fs]: Why do you think it's unnecessary?

File: src/fsharp/vs/ServiceUntypedParse.fs (hunk 2 above)
Comment (vasily-kirichenko) [inline src/fsharp/vs/ServiceUntypedParse.fs]: Same as above.

File: src/fsharp/vs/ServiceDeclarations.fs (hunk 1 above)
Comment (Phrohdoh) [inline src/fsharp/vs/ServiceDeclarations.fs]: This is a non-gain change. What does this file gain from this whitespace addition?
